lib/kubernetes-deploy/kubernetes_resource/deployment.rb

```diff
@@ -54,7 +54,8 @@ def failure_message
     def timeout_message
       progress_seconds = @definition['spec']['progressDeadlineSeconds']
       if progress_seconds
-        "Deploy timed out due to progressDeadlineSeconds of #{progress_seconds} seconds. #{@latest_rs&.timeout_message}"
+        "Deploy timed out due to progressDeadlineSeconds of #{progress_seconds} seconds, "\
+        " reason: #{@progress['reason']} #{@latest_rs&.timeout_message}"
```

Comment:
Since these lines are already long, let's put a line break in the message before the `@latest_rs` part.

Reply:
Done!